src/components/standalone/security/threat_shield/AllowlistTab.vue

```diff
@@ -0,0 +1,161 @@
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import { NeButton, NeInlineNotification, NeSkeleton } from '@nethesis/vue-components'
+import { NeEmptyState, getAxiosErrorMessage } from '@nethserver/vue-tailwind-lib'
+import { ref } from 'vue'
+import { useUciPendingChangesStore } from '@/stores/standalone/uciPendingChanges'
+import { onMounted } from 'vue'
+import { ubusCall } from '@/lib/standalone/ubus'
+import AllowlistTable from './AllowlistTable.vue'
+import DeleteAddressModal from './DeleteAddressModal.vue'
+import CreateOrEditAddressDrawer from './CreateOrEditAddressDrawer.vue'
+import { useNotificationsStore } from '@/stores/standalone/notifications'
+
+export type AllowlistAddress = {
+  address: string
+  description: string
+}
+
+const { t } = useI18n()
+const uciChangesStore = useUciPendingChangesStore()
+const notificationsStore = useNotificationsStore()
+
+const error = ref({
+  notificationDescription: '',
+  notificationDetails: ''
+})
+const fetchError = ref(false)
+const allowlist = ref<AllowlistAddress[]>([])
+const selectedAddress = ref<AllowlistAddress>()
+const loading = ref(false)
+const showDeleteAddressModal = ref(false)
+const showCreateOrEditAddressDrawer = ref(false)
+
+function openCreateEditAddressDrawer(itemToEdit?: AllowlistAddress) {
+  selectedAddress.value = itemToEdit
+  showCreateOrEditAddressDrawer.value = true
+}
+
+function openDeleteAddressModal(itemToDelete: AllowlistAddress) {
+  selectedAddress.value = itemToDelete
+  showDeleteAddressModal.value = true
+}
+
+async function fetchAllowlist() {
+  error.value.notificationDescription = ''
+  error.value.notificationDetails = ''
+
+  try {
+    loading.value = true
+    allowlist.value = (await ubusCall('ns.threatshield', 'list-allowed')).data.data
+  } catch (err: any) {
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+    error.value.notificationDetails = err.toString()
+    fetchError.value = true
```

Comment:
It seems like this variable is never reset to false, maybe we should do it before the `try`